suite-native/navigation/src/components/ScreenHeader.tsx

```diff
@@ -1,22 +1,24 @@
 import React, { ReactNode } from 'react';
 import { View } from 'react-native';
 
+import { RequireAllOrNone } from 'type-fest';
 import { useNavigation } from '@react-navigation/native';
 
 import { NativeStyleObject, prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { Box, IconButton, StepsProgressBar, Text } from '@suite-native/atoms';
+import { Box, IconButton, StepsProgressBar, Text, VStack } from '@suite-native/atoms';
 import { TypographyStyle } from '@trezor/theme';
 
 type ScreenHeaderWithIconsProps = {
     leftIcon?: ReactNode;
     rightIcon?: ReactNode;
-    title?: string;
+    content?: ReactNode;
```

Comment:
Maybe please use `string | ReactNode`. I know `string` is already in `ReactNode` but it will more explicitely say that if you provide string behavior will be different

Reply:
fixed in 82cd8639c